vumi_twilio_api/twilio_api.py

```diff
@@ -41,7 +47,20 @@ class TwilioAPIConfig(ApplicationWorker.CONFIG_CLASS):
     api_version = ConfigText(
         "The version of the API, used in the api URL",
         default="2010-04-01", static=True)
-    redis_manager = ConfigDict("Redis config.", static=True)
+    redis_manager = ConfigDict("Redis config.", required=True, static=True)
+    riak_manager = ConfigRiak("Riak config.", required=True, static=True)
+    client_path = ConfigText(
+        "The web path that the API worker should send requests to",
+        required=True, static=True)
+    client_method = ConfigText(
+        "The HTTP method that the API worker uses when sending requests",
+        default='POST', static=True)
+    status_callback_path = ConfigText(
+        "The web path that the API sends a request to when the call ends",
+        default=None, static=True)
+    status_callback_method = ConfigText(
+        "The HTTP method to use when sending the callback status",
+        default='POST', static=True)
```

Comment:
If I'm understanding the Twilio documentation correctly, all of these should just be the equivalent of account fallback values because they can also be specified in the `Call` request?